com.unity.visualeffectgraph/CHANGELOG.md

```diff
@@ -122,6 +122,7 @@ The version number for this package has increased due to a version update of a r
 - Fix culling of point output [Case 1225764](https://issuetracker.unity3d.com/product/unity/issues/guid/1225764/)
 - Compilation issue when normal is used in shadergraph for opacity with unlit output
 - Fix Exception on trying to invert a degenerate TRS matrix [Case 1307068](https://issuetracker.unity3d.com/product/unity/issues/guid/1307068/)
+- Fix bounds helper tool (automatic systems culling, world bounds computation, ...)
```

Comment:
It's unexpected to have this addition in previous version of changelog ⬆️ (not blocking neither), it's already in 12.1.5 section.
```suggestion
```